src/main/java/com/wimbli/WorldBorder/WorldTrimTask.java

```diff
@@ -28,8 +28,10 @@
 	private transient int taskID = -1;
 	private transient Player notifyPlayer = null;
 	private transient int chunksPerRun = 1;
+	private transient WorldFileDataType typeToDo = null;
```

Comment:
```suggestion
	private transient WorldFileDataType typeToTrim = null;
```
I'd prefer if this field was changed to this